src/Microsoft.ML.StandardLearners/FactorizationMachine/FieldAwareFactorizationMachineUtils.cs

```diff
@@ -130,12 +131,12 @@ public IRow GetRow(IRow input, Func<int, bool> predicate, out Action action)
                 getters[1] = probGetter;
             }
 
-            return new SimpleRow(Schema, input, getters);
+            return new SimpleRow(OutputSchema, input, getters);
         }
 
         public Func<int, bool> GetDependencies(Func<int, bool> predicate)
         {
-            if (Enumerable.Range(0, Schema.ColumnCount).Any(predicate))
+            if (Enumerable.Range(0, OutputSchema.ColumnCount).Any(predicate))
```

Comment:
>OutputSchema [](start = 36, length = 12)

InputSchema?

Reply:
No.  That means that "If any of output column is required, all input columns are needed".

---
In reply to: [237958980](https://github.com/dotnet/machinelearning/pull/1759#discussion_r237958980) [](ancestors = 237958980)